packages/expo-dev-client-components/src/theme.ts

```diff
@@ -138,15 +138,15 @@ export const text = {
   },
 
   weight: {
-    thin: { fontWeight: '100' as TextStyle['fontWeight'] },
-    extralight: { fontWeight: '200' as TextStyle['fontWeight'] },
-    light: { fontWeight: '300' as TextStyle['fontWeight'] },
-    normal: { fontWeight: '400' as TextStyle['fontWeight'] },
-    medium: { fontWeight: '500' as TextStyle['fontWeight'] },
-    semibold: { fontWeight: '600' as TextStyle['fontWeight'] },
-    bold: { fontWeight: '700' as TextStyle['fontWeight'] },
-    extrabold: { fontWeight: '800' as TextStyle['fontWeight'] },
-    black: { fontWeight: '900' as TextStyle['fontWeight'] },
+    thin: {  fontFamily: 'Inter-Thin'  },
+    extralight: { fontFamily: 'Inter-ExtraLight' },
+    light: {  fontFamily: "Inter-Light" },
+    normal: { fontFamily: "Inter-Regular" },
+    medium: { fontFamily: "Inter-Medium" },
+    semibold: { fontFamily: "Inter-SemiBold" },
+    bold: { fontFamily: "Inter-Bold" },
+    extrabold: { fontFamily: "Inter-ExtraBold" },
+    black: { fontFamily: "Inter-Black" },
```

Comment:
@FiberJW in case you wanted to see / have input on this

Reply:
this makes sense!